recursion/ex4_stringReverse.py

```diff
@@ -0,0 +1,16 @@
+# ex 4
+
+def stringReverse(str: str):
+    '''
+    A recursive function which reverses a string
+    '''
+    if len(str) == 0:
+        print("Error! please enter at least one character.")
```

Comment:
I do not understand this comment. Please rewrite your point.

Reply:
in English, This if means that if a user will insert an empty string, The function will print a message and wont give an answer.
In my opinion and because of the fact that I didn't define an empty string to be "Bad" i expect you to return a valid message for this input as well.
The output for an empty string "" is "".
In other words, the reverse of "" is "".

Reply:
What about this?

Reply:
Ive fixed this.